README.md

```diff
@@ -17,4 +18,19 @@ Look at [variables](./dags/openshift_nightlies/docs/variables.md) to see how var
 
 ## Installing Airflow
 
-Refer to the [Leviathan Repo](https://github.com/whitleykeith/leviathan)
\ No newline at end of file
+To install Airflow you simply need to fork the repo and run the following on a box that has access to an openshift cluster:
+
+```bash
+# all commands are run at the root of your git repo
+# install the airflow stack and have it point to your fork of the dag code.
+# USER should be your github username, and BRANCH should be the branch of that repo. 
+./scripts/install.sh -u $USER -b $BRANCH
```

Comment:
Maybe use the code local to the fork, this way we can skip on passing the USER and BRANCH.

Reply:
^^ as default and params if we want to use a different user/branch

Reply:
I'm fine with the idea but I haven't found a great way of doing it with the git config commands I've been messing around with. I've only been able to find a user's email or their human readable name like "Keith Whitley"